dash/development/base_component.py

```diff
@@ -196,7 +196,7 @@ def _set_random_id(self):
                 """
             )
 
-        v = str(uuid.UUID(int=rd.randint(0, 2 ** 128)))
+        v = str(uuid.UUID(int=rd.randint(0, 2**128)))
```

Comment:
:see_no_evil:

Reply:
Will have to look into whether we're linting correctly on CI - this was caught by the precommit hook for me 🤷